kube-runtime/src/controller/mod.rs

```diff
@@ -38,13 +38,13 @@ mod runner;
 
 #[derive(Debug, Error)]
 pub enum Error<ReconcilerErr: std::error::Error + 'static, QueueErr: std::error::Error + 'static> {
-    #[error("ObjectNotFound")]
+    #[error("tried to reconcile object {0} that was not found in local store")]
     ObjectNotFound(ObjectRef<DynamicObject>),
-    #[error("ReconcilerFailed: {0}")]
-    ReconcilerFailed(#[source] ReconcilerErr),
-    #[error("SchedulerDequeueFailed: {0}")]
+    #[error("reconciler for object {1} failed")]
+    ReconcilerFailed(#[source] ReconcilerErr, ObjectRef<DynamicObject>),
```

Comment:
This relies on the `Display` impl on `ObjectRef` right?

Reply:
Yep